api/nakadi-event-bus-api.yaml

```diff
@@ -414,8 +423,10 @@ paths:
 
         The event stream is formated as a sequence of `EventStreamBatch`es separated by `\n`. Each
         `EventStreamBatch` constains a chunk of Events and a `Cursor` pointing to the **end** of the
-        chunk (i.e. last delivered Event). Currently this format is the only one supported by the system,
-        but in the future other MIME types will be suported (like `application/event-stream`)
+        chunk (i.e. last delivered Event). The cursor might specify the offset with the symbolic value
+        `OLDEST`, which will open the stream starting from the oldest available offset in the partition.
```

Comment:
Not necessarily. Is more about having SOME name here and not XXX. The discussion on the name is to my understanding still open.